indico/modules/events/contributions/controllers/display.py

```diff
@@ -55,10 +55,10 @@ class RHContributionDisplayBase(RHDisplayEventBase):
 
     def _check_access(self):
         RHDisplayEventBase._check_access(self)
-        published = contribution_settings.get(self.event, 'published')
         if not self.contrib.can_access(session.user):
             raise Forbidden
-        if not published:
+        published = contribution_settings.get(self.event, 'published')
+        if not published and not self.event.can_manage(session.user):
```

Comment:
Can indico admins also access this `if` through `self.event.can_manage(session.user)` ?

Reply:
yes, admins always have access unless explicitly opted out